helper.php

```diff
@@ -25,9 +25,35 @@ public function getDB()
             return null;
         }
 
+        $this->registerUDF($sqlite);
+
         return $sqlite;
     }
 
+    /**
+     * Register user defined functions
+     *
+     * @param helper_plugin_sqlite $sqlite
+     */
+    protected function registerUDF($sqlite)
+    {
+        $sqlite->create_function('AUTH_ISMEMBER', [$this, 'auth_isMember'], -1);
+    }
+
+    /**
+     * Wrapper function for auth_isMember which accepts groups as string
+     *
+     * @param string $memberList
+     * @param string $user
+     * @param string $groups
+     * @return bool
+     */
+    public function auth_isMember($memberList, $user, $groups)
+    {
+        $a = 1;
+        return auth_isMember($memberList, $user, explode(',', $groups));
```

Comment:
I would not use commas as a separator. People use weird group and user names sometimes.

Reply:
I think `///` should be safe, right?